src/routes/wallets.js

```diff
@@ -0,0 +1,19 @@
+var express = require('express'),
+    wallets = require('../api').wallets;
+
+
+module.exports = express.Router()
+  .post('/', function(req, res, next) {
+    var conf = req.app.get('conf');
+
+    // TODO validate request body
+    // TODO handle errors given back from api
+    wallets.create({
+        wallet_server: conf.wallet_server,
+        kdf_params: conf.kdf_params,
+        username: req.body.username,
+        password: req.body.password
+      })
+      .then(function() { res.send({}); })
```

Comment:
Re the full username - the stellar network doesn't seem to have any concept of usernames itself. The wallet server seems to require emails to be given as usernames, but those usernames aren't given to the stellar network. The data generated by `util.generateKeyPair` contains an address field (which is derived from the public key), and that field is used to identify accounts in the stellar network. Will keep in mind that we might need to return the public key and address at some point.